### ISSUE TITLE
nb of watched tickets with unseen articles displayed on icon inconsistent with the number of displayed tickets when you click on this icon

### ISSUE THREAD

**Issue by gerald2545** [znuny/Znuny#372]
### Environment

- OS:           windows 10
- Browser:      firefox, chrome...
- Znuny version: 6.5.1

### Expected behaviour

be coherent with icon "number of watched tickets which have an new article" and the number of articles displayed when clicking on this icon
### Actual behaviour

currently, if the agent has more than 1000 watched tickets, the icon and displayed tickets are inconsistent

### How to reproduce

- be a watcher on more than 1000 tickets
- another agent/customer add articles on some of these tickets 

### Additional information

problem comes from /opt/otrs/Kernel/Modules/AgentTicketWatchView.pm
a variable is set to "my $Limit = 10_000;" but not used where it should have been used. Some limits are hard coded to 1_000

to fix this problem, juste replace 1_000 with $Limit: 
```
[root@esitoul-prod Modules]# diff -u AgentTicketWatchView.pm.ori AgentTicketWatchView.pm.ori.modified
--- AgentTicketWatchView.pm.ori 2023-01-03 09:35:24.138775436 +0100
+++ AgentTicketWatchView.pm.ori.modified 2023-01-03 09:38:35.453894167 +0100
@@ -311,7 +311,7 @@
             %{ $Filters{$Filter}->{Search} },
             %ColumnFilter,
             Result => 'ARRAY',
- Limit => 1_000,
+ Limit => $Limit,
         );
     }

@@ -340,7 +340,7 @@
             %{ $Filters{All}->{Search} },
             %ColumnFilter,
             Result => 'ARRAY',
- Limit => 1_000,
+ Limit => Limit,
         );

         my %ViewableTicketsNotNew;

```

### Screenshots
I corrected the error so no more screenshot can be done ...and my colleagues who had the problem is on vacation

**Comment by hanneshal** [comment]
Thanks for the report.
We put it on the list for the next release

Side note:
>currently, if the agent has more than 1000 watched tickets, the icon and displayed tickets are inconsistent

And the agent has a serious problem ;)

**Comment by gerald2545** [comment]
> And the agent has a serious problem ;)  

or we have a wrong usage of znuny ;(

**Comment by NiklasSchmitt** [comment]
Internal issue: 691

**Comment by rkaldung** [comment]
@gerald2545 This was fixed with the release of Znuny LTS 6.5.4 (ff3d784b3196f4c7dae0dc3f907e59e6a678cf8c) and Znuny 7.0.9 (81fcddde5b866a0d23d4a602ede73641196062f2).
